### PR TITLE
Adding man command to call the man page within machinery

### LINKED ISSUE #348: Man page not accessible when installing from gem
The gem installs the man page as part of its gem-specific files, but not at a location accessible by the `man` command by default. While this seems to be the standard practice for gems, it would be great, if there would be an easy way to access the man page.

I would suggest to implement a command `machinery man`, which shows the man page just as `man machinery` would, if the man page would be installed properly.

### PR DESCRIPTION
**Pull request by keberle94**
Fixes #348 
Squashes #1038

### CODE REVIEW

File: spec/unit/man_task_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      expect(LocalSystem).to receive(:validate_existence_of_package).with("man")
+      allow_any_instance_of(Kernel).to receive(:system).with("man", subject.man_path)
+
+      subject.man
+    end
+
+    it "calls the machinery man page" do
+      allow(LocalSystem).to receive(:validate_existence_of_package).with("man")
+      expect(subject).to receive(
+        "system").with("man", subject.man_path)
```
Comment (tgoettlicher) [inline spec/unit/man_task_spec.rb]: Wouldn't it read easier with the line break before `with`?
``expect(subject).to receive("system").
  with("man", subject.man_path)``

File: lib/cli.rb [hunk 2]
```diff
@@ -566,6 +565,18 @@ def self.parse_scopes(scope_string)
...
+    Shows the man page of the machinery tool.
+
+  LONGDESC
+  command :man do |c|
+    c.action do
+      task = ManTask.new
+      task.man
+    end
+  end
+
```
Comment (tgoettlicher) [inline lib/cli.rb]: Could you remove the superfluous blank lines?

File: man/machinery-man.1.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+## Man — Shows man page
```
Comment (tgoettlicher) [inline man/machinery-man.1.md]: I wasn't precise enough what I meant when I said it's a head line:  `## man — Shows Man Page`.

Comment (keberle94) [inline man/machinery-man.1.md]: Sorry haven't looked properly into it.

**Comment by keberle94** [comment]
Squashed by #1053

**Pull request closed without merging**